src/js/controllers/step1/LottoController.js

```diff
@@ -0,0 +1,219 @@
+import LottoTicket from "../../models/LottoTicket.js";
+import ConsoleInput from "../../views/step1/ConsoleInput.js";
+
+class LottoController {
+  #readline;
+  #consoleInput;
+  #money;
+  #winningCriteria = [
+    {
+      winningCount: 3,
+      hasToWinBonus: false,
+      winningAmount: 5_000,
+      numOfWinTicket: 0,
+    },
+    {
+      winningCount: 4,
+      hasToWinBonus: false,
+      winningAmount: 50_000,
+      numOfWinTicket: 0,
+    },
+    {
+      winningCount: 5,
+      hasToWinBonus: false,
+      winningAmount: 1_500_000,
+      numOfWinTicket: 0,
+    },
+    {
+      winningCount: 5,
+      hasToWinBonus: true,
+      winningAmount: 30_000_000,
+      numOfWinTicket: 0,
+    },
+    {
+      winningCount: 6,
+      hasToWinBonus: false,
+      winningAmount: 2_000_000_000,
+      numOfWinTicket: 0,
+    },
+  ];
+
+  constructor(readline) {
+    this.#readline = readline;
+    this.#consoleInput = new ConsoleInput(readline);
+  }
+
+  validateMoney(money) {
+    if (isNaN(money)) {
+      throw new Error("금액은 숫자 타입이어야 합니다.");
+    }
+
+    if (money < 1000) {
+      throw new Error("금액은 1000원 이상이어야 합니다.");
+    }
+
+    if (money % 1000 > 0) {
+      throw new Error("금액은 1000원 단위여야 합니다.");
+    }
+  }
+
+  validateNumber(number) {
+    if (isNaN(number)) throw new Error("번호는 숫자 타입이어야 합니다.");
+
+    if (number < 1 || number > 45) throw new Error("번호는 1 이상 45 이하 숫자여야 합니다.");
+  }
+
+  validateWinningNumbers(winningNumbers) {
+    winningNumbers.forEach((number) => {
+      this.validateNumber(number);
+    });
```

Comment:
소소하게 이렇게 쓸 수도 있을거 같네요
```suggestion
    winningNumbers.forEach(this.validateNumber);
```

Reply:
현재 `validateNumber` 메서드를 여러 군데에서 호출하고 있기 때문에 에러 상황이 발생했을 때 어디에서 에러가 발생한건지 탐지하기 어려울 수 있을거 같아요.

`Array.prototye.every()` 같은 메서드를 사용해서 `winningNumbers` 배열에 담겨있는 값들이 유효한지 확인하고 여기에서 에러 메세지를 던져주면 어떨까요?